.github/CODEOWNERS

```diff
@@ -231,6 +231,7 @@
 /pkg/collector/corechecks/embed/apm*.go            @Datadog/agent-platform @DataDog/agent-apm
 /pkg/collector/corechecks/embed/process_agent*.go  @Datadog/agent-platform @DataDog/processes
 /pkg/collector/corechecks/net/          @DataDog/agent-platform
+/pkg/collector/corechecks/oracle        @DataDog/database-monitoring
```

Comment:
We should also add `cmd/agent/dist/conf.d/oracle-dbm.d/conf.yaml.default` to be owned by  @DataDog/database-monitoring

Reply:
done.